#Importing Libraries 

In [1]:
#Import libraries
import pandas as pd
import numpy as np
import os

#5.Import the “orders.csv” data set into your notebook using the os library and the vars_list shortcut, omitting the “eval_set” column in your final dataframe.

In [2]:
path=r'C:\Users\Admin\Desktop\Project1'

In [3]:
df = pd.read_csv(r'C:\Users\Admin\Desktop\Project1\orders.csv', index_col = False)

In [4]:
vars_list = ['order_id', 'user_id', 'order_number', 'order_dow', 'order_hour_of_day', 'days_since_prior_order']

In [5]:
vars_list

['order_id',
 'user_id',
 'order_number',
 'order_dow',
 'order_hour_of_day',
 'days_since_prior_order']

#Print the first 5 rows of your dataframe

In [9]:
df.tail()

order_id  user_id eval_set  order_number  order_dow  \
3421078   2266710   206209    prior            10          5   
3421079   1854736   206209    prior            11          4   
3421080    626363   206209    prior            12          1   
3421081   2977660   206209    prior            13          1   
3421082    272231   206209    train            14          6   

         order_hour_of_day  days_since_prior_order  
3421078                 18                    29.0  
3421079                 10                    30.0  
3421080                 12                    18.0  
3421081                 12                     7.0  
3421082                 14                    30.0

#Import the “products.csv”

In [11]:
path=r'C:\Users\Admin\Desktop\Project1'

In [13]:
df_prods = pd.read_csv(r'C:\Users\Admin\Desktop\Project1\products.csv', index_col = False)

#Print the first 20 rows and last 35 rows of the df_prods dataframe.


In [14]:
df.tail(20)

product_id                                product_name  aisle_id  \
49673       49669                            K Cup Dark Blend       100   
49674       49670                         Beef Summer Sausage       106   
49675       49671                        Milk Chocolate Drops        45   
49676       49672                      Cafe Mocha K-Cup Packs        26   
49677       49673       Stone Baked Multi Grain Artisan Rolls       129   
49678       49674     Frozen Greek Yogurt Bars Chocolate Chip        37   
49679       49675         Cinnamon Dolce Keurig Brewed K Cups        26   
49680       49676                      Ultra Red Energy Drink        64   
49681       49677             Thick & Chunky Sloppy Joe Sauce        59   
49682       49678             Large Chicken & Cheese Taquitos       129   
49683       49679                     Famous Chocolate Wafers        61   
49684       49680          All Natural Creamy Caesar Dressing        89   
49685       49681     Spaghetti with Meatballs and Sauce Meal        38   
49686       49682                          California Limeade        98   
49687       49683                              Cucumber Kirby        83   
49688       49684   Vodka, Triple Distilled, Twist of Vanilla       124   
49689       49685          En Croute Roast Hazelnut Cranberry        42   
49690       49686                            Artisan Baguette       112   
49691       49687  Smartblend Healthy Metabolism Dry Cat Food        41   
49692       49688                      Fresh Foaming Cleanser        73   

       department_id  prices  
49673             21     4.7  
49674             12    19.2  
49675             19     3.0  
49676              7     6.5  
49677              1     5.6  
49678              1    11.1  
49679              7    14.0  
49680              7    14.5  
49681             15     8.9  
49682              1     3.4  
49683             19     6.0  
49684             13     4.9  
49685              1     6.9  
49686              7     4.3  
49687              4    13.2  
49688              5     5.3  
49689              1     3.1  
49690              3     7.8  
49691              8     4.7  
49692             11    13.5

In [15]:
df.tail(35)

product_id                                       product_name  \
49658       49654                   Teriyaki Sauce, Sesame, Original   
49659       49655                                        Apple Cider   
49660       49656                         Masada Kosher Pocket Bread   
49661       49657                                  Cabernet Tomatoes   
49662       49658                         Brie with Herbs Foil Wedge   
49663       49659                            Organic Creamed Coconut   
49664       49660                        Professionals Sleek Shampoo   
49665       49661                                              Porto   
49666       49662                       Bacon Cheddar Pretzel Pieces   
49667       49663  Ultra Protein Power Crunch Peanut Butter N' Ho...   
49668       49664                     Lemon Cayenne Drinking Vinegar   
49669       49665      Super Dark Coconut Ash & Banana Chocolate Bar   
49670       49666                      Ginger Snaps Snacking Cookies   
49671       49667           Enchilada with Spanish Rice & Beans Meal   
49672       49668                     Apple Cinnamon Scented Candles   
49673       49669                                   K Cup Dark Blend   
49674       49670                                Beef Summer Sausage   
49675       49671                               Milk Chocolate Drops   
49676       49672                             Cafe Mocha K-Cup Packs   
49677       49673              Stone Baked Multi Grain Artisan Rolls   
49678       49674            Frozen Greek Yogurt Bars Chocolate Chip   
49679       49675                Cinnamon Dolce Keurig Brewed K Cups   
49680       49676                             Ultra Red Energy Drink   
49681       49677                    Thick & Chunky Sloppy Joe Sauce   
49682       49678                    Large Chicken & Cheese Taquitos   
49683       49679                            Famous Chocolate Wafers   
49684       49680                 All Natural Creamy Caesar Dressing   
49685       49681            Spaghetti with Meatballs and Sauce Meal   
49686       49682                                 California Limeade   
49687       49683                                     Cucumber Kirby   
49688       49684          Vodka, Triple Distilled, Twist of Vanilla   
49689       49685                 En Croute Roast Hazelnut Cranberry   
49690       49686                                   Artisan Baguette   
49691       49687         Smartblend Healthy Metabolism Dry Cat Food   
49692       49688                             Fresh Foaming Cleanser   

       aisle_id  department_id  prices  
49658         5             13     4.0  
49659        98              7    10.7  
49660       128              3     7.1  
49661        83              4     8.3  
49662         2             16     3.9  
49663        17             13     3.1  
49664        22             11     6.7  
49665       134              5     8.2  
49666       107             19     3.6  
49667        57             14    10.2  
49668       100             21    13.7  
49669        45             19     6.9  
49670        61             19     5.2  
49671        38              1     6.6  
49672       101             17     5.6  
49673       100             21     4.7  
49674       106             12    19.2  
49675        45             19     3.0  
49676        26              7     6.5  
49677       129              1     5.6  
49678        37              1    11.1  
49679        26              7    14.0  
49680        64              7    14.5  
49681        59             15     8.9  
49682       129              1     3.4  
49683        61             19     6.0  
49684        89             13     4.9  
49685        38              1     6.9  
49686        98              7     4.3  
49687        83              4    13.2  
49688       124              5     5.3  
49689        42              1     3.1  
49690       112              3     7.8  
49691        41     

In [ ]:
#9.Print the names of the columns in the df_prods dataframe.

In [16]:
df_prods.columns

Index(['product_id', 'product_name', 'aisle_id', 'department_id', 'prices'], dtype='object')

In [ ]:
#10.Print the number of rows and columns in the df_prods dataframe.

In [18]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 49693 entries, 0 to 49692
Data columns (total 5 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   product_id     49693 non-null  int64  
 1   product_name   49677 non-null  object 
 2   aisle_id       49693 non-null  int64  
 3   department_id  49693 non-null  int64  
 4   prices         49693 non-null  float64
dtypes: float64(1), int64(3), object(1)
memory usage: 1.9+ MB


In [17]:
df.head()

product_id                                       product_name  aisle_id  \
0           1                         Chocolate Sandwich Cookies        61   
1           2                                   All-Seasons Salt       104   
2           3               Robust Golden Unsweetened Oolong Tea        94   
3           4  Smart Ones Classic Favorites Mini Rigatoni Wit...        38   
4           5                          Green Chile Anytime Sauce         5   

   department_id  prices  
0             19     5.8  
1             13     9.3  
2              7     4.5  
3              1    10.5  
4             13     4.3

#11a)What’s the maximum value of the “aisle_id” column?

It is 104


#11b)What’s the data type of the “department_id” column?

it is INTEGER